src/fonduer/supervision/labeler.py

```diff
@@ -37,6 +37,11 @@
 
 logger = logging.getLogger(__name__)
 
+# Snorkel standardized ABSTAIN as follows:
+# ABSTAIN is None (LFs), -1 (label matrix), or 0 (LFAnalysis)
```

Comment:
This sentence is not clear.

Reply:
How about this?
# Snorkel changed the label conventions: ABSTAIN is now represented by -1 (used to be 0)
# Details can be found at https://github.com/snorkel-team/snorkel/pull/1309

Reply:
Looks good! Also, add 0 index for classification task labels.

Reply:
See the change at 6428ddf8ea5df85a4168455173036f705e3be2c5.